Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,44 @@
+
+## BUILDABLE SCHEMATIC OF SUBSYSTEM
+Shown is the pseudo-code for the operating code.  The final form of the code may have parts added and additional functionalities included, but this is the general idea of how the code should flow.  Proceeding from each section is assuming that there are no errors detected indicating a “Success”.  If an error arises, a default handler will need to be called to assess the situation.  It should also be noted that the loop is intended to be theoretically infinite, however, there are explicit and implicit breaks at the battery running critically low (explicit) and the shutoff button being pushed (implicit).  “Timers” refer to either physical hardware or coded timers that go off to alert the system that the sensor(s) sampling is ready.
+
+![pseudo code 2](https://user-images.githubusercontent.com/110966922/202583040-c28e27fe-c175-4287-a8cb-cefafbfe327b.png)
+
+Shown is a schematic of the Arduino Mega 2560 Rev 3 and USB Host Shield:
+![CAPSTONE 1 Microcontroller Schematic Updated](https://user-images.githubusercontent.com/118767661/206057697-6be6d265-4ac2-411e-a632-a95b974fea32.png)
+
+## ANALYSIS OF CHOSEN COMPONENT(S)
+The team has chosen to look at the Arduino Mega 2560 Rev 3 and Arduino USB Shield Host as the solutions.  These components, in conjunction, will satisfy the constraints and jobs of this subsystem.  Following are the corresponding fulfillment of the constraints:
+
+1. The Arduino Mega 2560 Rev 3 falls under the preferred maximum of $60 at a current price of $48.40 (as of 11-8-22).  The USB Host Shield component has a price of $20.99 as of the same date.   
+2. 4 UART serial hardware ports are present on the board.  SPI is supported as well with included libraries.  These enable the Mega to issue instructions and exchange data between other components and subsystems (Communication).
+3. The USB Host Shield enables the use of USB devices such as a Flash drive for external storage and communication. The Arduino Mega does not support USB as a standalone component.
+4. There are 54 digital input/output pins (of which 15 are PWM outputs) and 16 analog inputs.  This meets the needs for at least 18 digital (15 for sensors and 3 for USB Host) and 16 analog input pins.  There are also 4 UART pins and 21 digital (3 for SPI if chosen and 2 for battery monitoring discretely) may be assigned special purposes later on in design.  1 analog pin may also be used for taking a continuous measurement for the battery monitoring.
+5. The code is currently of an unknown size, so it is necessary to aim slightly higher in terms of needed processing memory.  The Arduino Mega has 256 KB of Flash, 8 KB of SRAM, and 4 KB of EEPROM.  The SRAM and Flash memory are satisfactory for the subsystem’s needs.
```

Comment:
First sentence is not needed here. The constraints already list required memory size.

Reply:
Removed.